L7-JUnit Params, Mockito/L7-Zad.7-Training/src/test/java/pl/babiarz/TrainingEffectivenessTest.java

```diff
@@ -0,0 +1,60 @@
+package pl.babiarz;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TrainingEffectivenessTest {
+    TrainingEffectiveness trainingEffectiveness;
+    TrainingLength trainingLength;
+    CaloriesBurned caloriesBurned;
+    AveragePulseRate averagePulseRate;
+
+    @BeforeEach
+    void setup() {
+        trainingEffectiveness = new TrainingEffectiveness();
+        trainingLength = new TrainingLength();
+        caloriesBurned = new CaloriesBurned();
+        averagePulseRate = new AveragePulseRate();
```

Comment:
Po co nam to skoro nic z tymi klasami nie robimy? ;)
Przydałyby się testy tych klas (TrainingLength, CaloriesBurner,AveragePulseRate)

Aktualnie testujemy je integracyjnie (wykorzystując trainingEffectiveness), ale to trochę słabe podejście. Minimum trzeba je również przetestować osobno.